### DISCUSSION TITLE
Are official nestybox images still mantained?

### DISCUSSION THREAD

**Discussion by cruizba** [nestybox/sysbox#764] [Q&A] [answered]
I see official images are not updated since a year ago:
https://hub.docker.com/u/nestybox

 Are official images still maintained?

**Comment by ctalledo** [comment, marked as answer]
Hi @cruizba,

> Are official images still maintained?

These images are meant as examples for folks to try, not to be used in production. As such, they are maintained but on a slower cadence. BTW, you can find the Dockerfiles for them [here](https://github.com/nestybox/dockerfiles/tree/master).

**Comment by cruizba** [comment]
Ok! Thank you for your answer :)
